networks/p2p/discover/node_test.go

```diff
@@ -239,7 +239,7 @@ func TestParseNode(t *testing.T) {
 			if err == nil {
 				t.Errorf("test %q:\n  got nil error, expected %#q", test.rawurl, test.wantError)
 				continue
-			} else if err.Error() != test.wantError {
```

Comment:
Does the error change depending on the version? Is there a way to predict, read, and compare the errors that should actually occur?